addon/components/bootstrap/-checkbox.js

```diff
@@ -0,0 +1,13 @@
+import Ember from 'ember';
+
+export default Ember.Checkbox.extend({
+  classNames: ['form-control'],
+
+  click() {
+  	const onClick = this.get('onClick');
```

Comment:
Guess the previous comment wasn't very clear. The documentation (and prior versions) used `onclick`, we should keep this name so we aren't breaking other stuff, so this should be `this.get('onclick')`.